### PR TITLE
OP-1179 Change DailyRollingFileAppender to RollingFileAppender

### PR DESCRIPTION
**Pull request by mwithi**
See OP-1179.

### CODE REVIEW

File: src/main/java/org/isf/utils/log/LogUtil.java [hunk 1]
```diff
@@ -43,8 +43,8 @@ public static String getLogFileAbsolutePath() {
 		Logger rootLogger = Logger.getRootLogger();
 		Appender appender = rootLogger.getAppender("RollingFile");
 
-		if (appender instanceof DailyRollingFileAppender) {
-			DailyRollingFileAppender rollingFileAppender = (DailyRollingFileAppender) appender;
+		if (appender instanceof RollingFileAppender) {
+			RollingFileAppender rollingFileAppender = (RollingFileAppender) appender;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/log/LogUtil.java:47]:
```suggestion
		if (appender instanceof RollingFileAppender rollingFileAppender) {	
```
Lines 46 and 47 can be replaced with this single line.   This is called a "Pattern variable" that was introduced in Java 16.

File: src/main/java/org/isf/utils/log/LogUtil.java [hunk 2]
```diff
@@ -43,8 +43,8 @@ public static String getLogFileAbsolutePath() {
...
 			String relativePath = rollingFileAppender.getFile();
 			String absolutePath = new File(relativePath).getAbsolutePath();
 			return absolutePath;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/utils/log/LogUtil.java:50]: I don't see the use in assigning the `new File()...` to a variable and then returning the variable.   Why not just return `new File()...`?

**Review by dbmalkovsky — COMMENTED** [review]
Left some comments.....

File: src/main/java/org/isf/utils/log/LogUtil.java (hunk 2 above)
Comment (mwithi) [reply src/main/java/org/isf/utils/log/LogUtil.java]: Just the result of many rework... done, thanks!

**Review by dbmalkovsky — COMMENTED** [review]
LGTM; thanks

**Pull request merged**